src/main/java/jalau/at18/kingoftokyo/view/GameFrame.java

```diff
@@ -0,0 +1,87 @@
+package jalau.at18.kingoftokyo.view;
+
+import jalau.at18.kingoftokyo.DiceFace;
+import jalau.at18.kingoftokyo.DiceRoller;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameFrame extends JFrame {
+    private static final int DEFAULT_POS = 30;
+    private static final int DEFAULT_WIDTH = 1200;
+    private static final int DEFAULT_HEIGHT = 700;
+
+    private static final int DEFAULT_LABEL_POSX = 800;
+    private static final int DEFAULT_LABEL_POSY = 500;
+    private static final int DEFAULT_LABEL_WIDTH = 20;
+    private static final int DEFAULT_LABEL_HEIGHT = 20;
+    private static final int INCREMENT_SPACE_LABEL = 20;
+    private static final int QUANTITY_DICES = 6;
+    private RollerButton rollerButton = new RollerButton();
+    private List<DiceFaceLabel> listDiceFaceLabel = new ArrayList<>();
+
+
+    public GameFrame() {
+        setResizable(false);
+        int count = 1;
+        int posX = DEFAULT_LABEL_POSX;
+        for (int index = 1; index <= QUANTITY_DICES; index++) {
+            DiceFaceLabel diceFaceLabel = new DiceFaceLabel(String.valueOf(count), posX,
+                    DEFAULT_LABEL_POSY, DEFAULT_LABEL_WIDTH, DEFAULT_LABEL_HEIGHT);
+            listDiceFaceLabel.add(diceFaceLabel);
+            add(diceFaceLabel);
+            count++;
+            posX = posX + INCREMENT_SPACE_LABEL;
+        }
+        add(rollerButton);
+
+
+        rollerButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                DiceRoller diceRoller = new DiceRoller();
+                List<DiceFace> listDiceFace = new ArrayList<>();
+                listDiceFace = diceRoller.randomDice(QUANTITY_DICES);
+                int count = 0;
+                for (DiceFaceLabel diceFaceLabel : listDiceFaceLabel) {
```

Comment:
This can be delegated to a UI component itself, check this example: https://github.com/fjprog101/at18/pull/175/files#diff-6c5abc7691978b8a5032e2ad3a544bad321ff148f89ad2527721c78e4ff3d3e5

Actually, try adding a controller so it creates the communication between the UI and model code

Reply:
Hi Jose, this code was done before we learned about  Controller and Model that is part of the MVC pattern arquitecture.  
I uploaded the code for sharing with my teammates after our meeting yesterday. 
Currently, the code implemented allows roll dice to display the random dice, including the icons by each DiceFace.

According to you suggestion from today , I will open another ticket to refactoring and carried this code to MVC.